### DISCUSSION TITLE
Websocket connection failed and live speech recognition issue

### DISCUSSION THREAD

**Discussion by AcastaPaloma** [deepgram/community#469] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

I followed the simple tutorial here: https://deepgram.com/learn/live-transcription-flask
I am also using flask. I'm fairly new to web development.

I've set up a simple page where I display a dummy text (here in German) and I prompt the user to read this text. After the API treats with the live speech sent to it, it sends back the words and I use a JavaScript block of code to compare the text and what was returned (so basically, what was said by the user). However, whenever I run this website from my local machine, this error keeps popping up:

WebSocket connection to 'ws://localhost:8080/listen' failed:

I've made sure that it matches my port number.

Can anyone find what I'm missing? Also, can anyone suggest a better way to compare the words pronounced by a user and the text displayed on the screen? My way seems half working, as it treats every single request one by one, I think. 

### If possible, please attach your code or paste it into the text box.

[question1.txt](https://github.com/deepgram/community/files/13507598/question1.txt)
[question2.txt](https://github.com/deepgram/community/files/13507600/question2.txt)
 the first file contains my relevant python code 
 and the second file contains the relevant html code
 
 
 so again, can anyone hekp me fix my two issues?
thank you so much

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@AcastaPaloma  Sorry for the very late reply here. but the project you mentioned is very outdated and isn't a project that is actively maintained any more. I'd suggest looking at our Starter App for Live Transcription with Flask.

https://github.com/deepgram-starters/live-flask-starter
